docs/release-notes/NuGet-6.13.md

```diff
@@ -23,6 +16,104 @@ NuGet distribution vehicles:
 
 <sup>1</sup> Installed with Visual Studio 2022 with any .NET workload
 
-## Not yet released
+## Summary: What's New in 6.13
+
+* Detect if restore used NuGetAudit or not for PackageReference projects - [#13778](https://github.com/NuGet/Home/issues/13778)
+
+* Support for credential providers deployed via .NET tools - [#12567](https://github.com/NuGet/Home/issues/12567)
+
+### Breaking changes
+
+* Support "Supplied by Platform" scenario in restore - [#7344](https://github.com/NuGet/Home/issues/7344)
```

Comment:
This is not enabled in this release, so let's put this in `What's new`. 
I'll do a follow up for ensuring the pruning is enabled is documented.